### PR TITLE
Added example file, updated .gitignore and README.

### PR DESCRIPTION
**Pull request by dhsrocha**
Commit for attending #48

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=51) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=51) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/php-http-client?pullRequest=51) it.</sub>

File: README.md [hunk 1]
```diff
@@ -138,6 +138,8 @@ print $response->body();
 
 - [Example Code](https://github.com/sendgrid/php-http-client/tree/master/examples)
 
+You can use the ```.env_example``` file located at "examples" folder. Just copy it to a ```.env``` folder ou can put on your project's root.
```
Comment (mbernier) [inline README.md]: can you please fix `ou` to `you` ?

**Comment by mbernier** [comment]
Checking to see if you can make this change?

**Comment by thinkingserious** [comment]
Hello @dhsrocha,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**